package.json

```diff
@@ -26,15 +26,14 @@
     "url": "https://github.com/Cimpress-MCP/AT-Exceptions/issues"
   },
   "homepage": "https://github.com/Cimpress-MCP/AT-Exceptions#readme",
-  "dependencies": {
-  },
-  "peerDependencies": {
-    "aws-sdk": "^2.0.0"
-  },
+  "dependencies": {},
+  "peerDependencies": {},
```

Comment:
Not sure why it ever peer depended on aws-sdk. I've removed that.

Reply:
Since I know who did it, copy&pasta from open API :P

Reply:
I am going to remove it from there as well. The only place we need it is in Authorizer and AT-TokenProvider because they call AWS SDKs. 
It looks like it spread from those as Corona would.